runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -130,7 +130,7 @@
 	DIARY_KANDARIN_ELITE(4478),
 
 	DIARY_KARAMJA_EASY(3578),
-	DIARY_KARAMJA_MEDIUM(3599),
+	DIARY_KARAMJA_MEDIUM(3598),	// Could also be 3599 (original value), 3589 (set on completion but without accepting rewards)
```

Comment:
I can confirm that 3598 is correct as it matches the index used [in the diary completion procedure](https://github.com/RuneStar/cs2-scripts/blob/b0633f0bc05802a2381a8af9e3d103215b338483/scripts/%5Bproc%2Cdiary_completion_info%5D.cs2).

Karamja is case 0, and the return value is an array/vector of size 12, essentially being four sets of 3, where the values are `( # of tasks in tier completed, # of tasks to complete in tier, tier completed:[0,1] )`. varbit 3598 is used to represent the medium tier completion there, so we can rely on that value being correct.